common/components/controllers/SignedUpController.jsx

```diff
@@ -12,6 +13,11 @@ class SignedUpController extends React.Component<{||}> {
   render(): React$Node {
     // TODO: Give better indication that email has been sent when user clicks resend link
     return (
+      <React.Fragment>
+      <Headers
+      title="Sign Up| DemocracyLab"
```

Comment:
Actually this should go in SignUpController.jsx.  This file is for the page that appears after a user signs up.

Reply:
I added a new commit to address the requested changes